docs/debug_logging.md

```diff
@@ -0,0 +1,54 @@
+Debug Logging
+=============
+
+OkHttp has internal APIs to enable debug logging. It uses the `java.util.logging` API which can be
+tricky to configure. As a shortcut, you can paste [OkHttpDebugLogging.kt]. Then enable debug logging
+for whichever features you need:
+
```

Comment:
Probably worth a sentence on Android, example logcat filter etc?

Reply:
Added. Turns out hooking up logcat to java.util.Logging nicely is a bit of a pain if you want to use `FINE`!

Reply:
(Good idea though!)